### ISSUE TITLE
make it running on windows

### ISSUE THREAD

**Issue by arademaker** [own-pt/sensetion.el#148]
possible changing the default backend of `request.el`

**Comment by odanoburu** [comment]
> possible changing the default backend of request.el

did you have any problems with it? `request.el` will use `url.el` if `curl` is not available, so it should be portable.

**Comment by hmuniz** [comment]
Yes, `request.el` will use `url.el` if `curl` is not available but in windows is a curl version avaible but we cannot trust it. So it is important windows users customize the variable `request-backend `.
